### PR TITLE
Validation: Contact Page 

### PR DESCRIPTION
**Pull request by jzhzhang66**
- Added error handling for contact page and form clearing when there is a successful submission
- utils.js: added some reusable constants for form submissions 
- Note: Still working on clearing textarea please let me know if you guys know any options!

### CODE REVIEW

File: components/text-input-field/TextInputField.style.js [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+  position: absolute;
+  color: red;
+  font-size: 0.7em;
+  margin: 0.5em 0 0 0;
+  text-shadow: 0.5px 0 #ff0000;
+  letter-spacing: 0.05em;
 `;
 
 const RedStar = styled.img`
   width: 2%;
```
Comment (isabellberzsenyi) [inline components/text-input-field/TextInputField.style.js:23]: I know it wasn't in this PR but the red star looks slightly too large on the desktop sizes

File: components/contact-page/ContactPage.styles.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import styled from 'styled-components';
 import { min } from '../../lib/responsive';
-
```
Comment (isabellberzsenyi) [inline components/contact-page/ContactPage.styles.js]: nit picky but can we not remove this line, for readability

File: components/contact-page/contact-form/ContactForm.js [hunk 3]
```diff
@@ -3,38 +3,104 @@ import TextInputBox from '../../text-input-box/TextInputBox';
...
   ContactFormContainer,
-  InputFieldWrapper,
   InputBoxWrapper,
   ButtonWrapper,
 } from './ContactForm.styles';
+import React, { useState } from 'react';
+import { emailEndpoint, axiosConfig, proxyurl } from '../../../lib/utils';
 
 export default function ContactForm({ togglePopup }) {
+  const axios = require('axios');
```
Comment (isabellberzsenyi) [inline components/contact-page/contact-form/ContactForm.js]: looks great!

File: components/text-input-field/TextInputField.style.js [hunk 4]
```diff
@@ -24,12 +38,24 @@ const TextInput = styled(P).attrs({
   display: block;
   border: none;
   background-color: ${colors.CREME};
-  border-bottom: 1px solid ${colors.BROWN};
+  border-bottom: ${(props) =>
+    props.showError ? `1px solid red` : `1px solid ${colors.BROWN}`};
```
Comment (isabellberzsenyi) [inline components/text-input-field/TextInputField.style.js]: let's use the `${colors.RED}` here

**Review by isabellberzsenyi — CHANGES_REQUESTED** [review]
looks great, just a couple minor comments!

File: components/text-input-field/TextInputField.style.js [hunk 5]
```diff
@@ -1,16 +1,30 @@
...
+  text-shadow: 0.5px 0 #ff0000;
+  letter-spacing: 0.05em;
 `;
 
 const RedStar = styled.img`
   width: 2%;
   margin-left: 0.5em;
+  @media ${min.desktopLg} {
+    width: 0.7em;
+  }
```
Comment (jzhzhang66) [inline components/text-input-field/TextInputField.style.js:27]: @isabellberzsenyi I think I fixed the star problem here?

**Review by isabellberzsenyi — APPROVED** [review]
LGTM

File: components/contact-page/contact-form/ContactForm.js (hunk 3 above)
Comment (jean-zhang) [inline components/contact-page/contact-form/ContactForm.js]: I would require axios outside of the component

**Review by josephmannis — APPROVED** [review]
Nice! This looks awesome

**Pull request merged**